src/cascadia/TerminalApp/TerminalTab.cpp

```diff
@@ -1587,6 +1587,70 @@ namespace winrt::TerminalApp::implementation
         });
     }
 
+    // Method Description:
+    // - Enable read-only mode on the active pane
+    // - If a parent pane is selected, this will ensure that all children have
+    //   the same read-only status.
+    void TerminalTab::EnablePaneReadOnly()

```

Comment:
I think we should replace these two functions with a single `TerminalTab::SetPaneReadOnly(bool)` or similarly named function. Because if anything we should probably also replace `control`'s `EnableReadOnly()` and `DisableReadOnly()` with just `SetReadOnly(bool)`.